src/app.js

```diff
@@ -44,5 +43,14 @@ app.use(express.static(path.join(__dirname, '..', 'public')));
 // config route controller
 app.use(controllers);
 
+// production error handler
+const HTTP_SERVER_ERROR = 500;
+app.use((err, req, res, next) => {
```

Comment:
Error handling middleware is already defined in `src/controllers/index.js` so we do not need it here